### ISSUE TITLE
Error in 2.2126.10 version of whatsapp web

### ISSUE THREAD

**Issue by alpires** [open-wa/wa-automate-nodejs#1860]
Error in new version of whatsapp web: "webpackChunkbuild is not defined" in wapi.js file

**Comment by marcogaruthix** [comment]
I am getting the error message "Session data most likely expired due to manual host account logout. Please re-authenticate this session." when I call create() function with no config.

**Comment by bernardoely** [comment]
My code stopped working today, I'm getting the error "webpackChunkbuild is not defined" too, please help!

**Comment by mahmoud-hassan-911** [comment]
change "webpackChunkbuild" to "webpackChunkwhatsapp_web_client"

**Comment by a7son** [comment]
same here
it keep crashing and saying..
 Session data most likely expired due to manual host account logout. Please re-authenticate this session.

**Comment by ardzz** [comment]
same here, I thought my session was expired but I tried relogin the app error and say `Session data most likely expired due to manual host account logout. Please re-authenticate this session.`

**Comment by a7son** [comment]
> change "webpackChunkbuild" to "webpackChunkwhatsapp_web_client"

where should i change it?

**Comment by wlskcode** [comment]
Yep... I'm also getting `Session data most likely expired due to manual host account logout. Please re-authenticate this session.`

**Comment by kmirea** [comment]
I just got this error as well

**Comment by ekoeryanto** [comment]
I am just migrating here yesterday. 😢

**Comment by manojsitapara** [comment]
I am also facing same issue 

> Session data most likely expired due to manual host account logout. Please re-authenticate this session.

**Comment by oAnjinho** [comment]
Same here
Session data most likely expired due to manual host account logout. Please re-authenticate

**Comment by wheezyy** [comment]
> change "webpackChunkbuild" to "webpackChunkwhatsapp_web_client"

Din't work for  me. Any other chans to make?

**Comment by deriknyvo** [comment]
I aways have the return "Session data most likely expired due to manual host account logout. Please re-authenticate this session" at start script

**Comment by smashah** [comment]
hi everyone, if you need this to be fixed asap then please find ` const canInjectEarly` line in `initializer.js` and change it to

```javascript
  const canInjectEarly = yield waPage.evaluate(() => { if (window.webpackChunkwhatsapp_web_client)window.webpackChunkbuild = window.webpackChunkwhatsapp_web_client; return (typeof webpackChunkbuild !== "undefined"); });
```

The next build will fix this issue but is currently stuck in the CI due to stupid TS warnings.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci minor`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.8.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.7.1...4.8.0) (2021-07-09)

- build(deps-dev): bump @types/express from 4.17.12 to 4.17.13 [`#1850`](https://github.com/open-wa/wa-automate-nodejs/pull/1850)
- build(deps): bump uuid-apikey from 1.5.2 to 1.5.3 [`#1846`](https://github.com/open-wa/wa-automate-nodejs/pull/1846)
- build(deps-dev): bump husky from 6.0.0 to 7.0.1 [`#1844`](https://github.com/open-wa/wa-automate-nodejs/pull/1844)
- build(deps): bump swagger-ui-dist from 3.51.0 to 3.51.1 [`#1834`](https://github.com/open-wa/wa-automate-nodejs/pull/1834)
- build(deps): bump puppeteer from 10.0.0 to 10.1.0 [`#1831`](https://github.com/open-wa/wa-automate-nodejs/pull/1831)
- build(deps-dev): bump @typescript-eslint/eslint-plugin from 4.28.0 to 4.28.2 [`#1841`](https://github.com/open-wa/wa-automate-nodejs/pull/1841)
- build(deps): bump ts-json-schema-generator from 0.93.0 to 0.94.0 [`#1845`](https://github.com/open-wa/wa-automate-nodejs/pull/1845)
- build(deps): bump rxjs from 7.1.0 to 7.2.0 [`#1843`](https://github.com/open-wa/wa-automate-nodejs/pull/1843)
- build(deps): bump parse-url from 5.0.6 to 5.0.7 [`#1830`](https://github.com/open-wa/wa-automate-nodejs/pull/1830)
- build(deps-dev): bump husky from 6.0.0 to 7.0.0 [`#1835`](https://github.com/open-wa/wa-automate-nodejs/pull/1835)
- build(deps-dev): bump @types/node from 15.6.1 to 16.0.0 [`#1840`](https://github.com/open-wa/wa-automate-nodejs/pull/1840)
- build(deps): bump swagger-stats from 0.99.1 to 0.99.2 [`#1838`](https://github.com/open-wa/wa-automate-nodejs/pull/1838)
- build(deps-dev): bump eslint from 7.29.0 to 7.30.0 [`#1839`](https://github.com/open-wa/wa-automate-nodejs/pull/1839)
- build(deps-dev): bump @typescript-eslint/parser from 4.28.0 to 4.28.2 [`#1842`](https://github.com/open-wa/wa-automate-nodejs/pull/1842)
- build(deps-dev): bump typedoc from 0.21.0 to 0.21.2 [`#1825`](https://github.com/open-wa/wa-automate-nodejs/pull/1825)
- build(deps-dev): bump marked from 2.1.2 to 2.1.3 [`#1826`](https://github.com/open-wa/wa-automate-nodejs/pull/1826)
- build(deps-dev): bump prettier from 2.3.1 to 2.3.2 [`#1823`](https://github.com/open-wa/wa-automate-nodejs/pull/1823)
- build(deps): bump parse-url from 5.0.5 to 5.0.6 [`#1824`](https://github.com/open-wa/wa-automate-nodejs/pull/1824)
- fix #1860 [`#1860`](https://github.com/open-wa/wa-automate-nodejs/issues/1860)
- 🥚  patches update. Fixes #1809 [`#1809`](https://github.com/open-wa/wa-automate-nodejs/issues/1809)
- 🥚 patches update, should fix #1819 [`#1819`](https://github.com/open-wa/wa-automate-nodejs/issues/1819)
- Release 4.8.0 [`86859d3`](https://github.com/open-wa/wa-automate-nodejs/commit/86859d32121c4ee3e30dbd94bf03f84744845e4f)
- 📝  chore: remove luminati/brightdata links #1821 [`a0d9a38`](https://github.com/open-wa/wa-automate-nodejs/commit/a0d9a38564beac3a2f240d1ff3620066abc595c9)
- 🏷️ ignore ts overload warning [`f175df3`](https://github.com/open-wa/wa-automate-nodejs/commit/f175df31c4f6c56dbc92f7e3f13cf7c791b5a07e)
- 🥅 ignore ts errors from express [`5b39d96`](https://github.com/open-wa/wa-automate-nodejs/commit/5b39d963a77e478a9430c58a0ce424ca9bbb72d4)
- 🥚 patches update [`e79a4bf`](https://github.com/open-wa/wa-automate-nodejs/commit/e79a4bf3c1189891b8be75f75d6a23288ea8e03a)
- 🥚 patches update [`ee6455b`](https://github.com/open-wa/wa-automate-nodejs/commit/ee6455b7365e4a7a3e919e7d9d5dd71f3e223571)
- 🥚 patches update [`66683f8`](https://github.com/open-wa/wa-automate-nodejs/commit/66683f8ceb71884cb6301327127475bb78d40a8a)
- 🥚 patches update [`f7ad9f5`](https://github.com/open-wa/wa-automate-nodejs/commit/f7ad9f567c1b5670d73edfeb56615c1d41f73a47)
- 🥚 patches update [`7d4fa3d`](https://github.com/open-wa/wa-automate-nodejs/commit/7d4fa3df33edb89a39f6f56a25d42d97e740b215)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.8.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.8.0...4.8.1) (2021-07-09)

- Release 4.8.1 [`4abb10c`](https://github.com/open-wa/wa-automate-nodejs/commit/4abb10cb72d48b4589a660beefeeba99360b2162)
- 🔥 remove `sessionDataInvalid` check as it causes false negative #1860 [`e6664ec`](https://github.com/open-wa/wa-automate-nodejs/commit/e6664ec5785339b3243ea41a7a6a3a6d6f8dccf9)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.8.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.8.1...4.8.2) (2021-07-09)

- Release 4.8.2 [`b35958c`](https://github.com/open-wa/wa-automate-nodejs/commit/b35958c54dc80f6f249923757967bb025379db31)
- 🐛 fixed logout detection, fix `getQrPng` undefined error #1860 [`721da48`](https://github.com/open-wa/wa-automate-nodejs/commit/721da4855e7147cdff8110dd8e5b4755d8ed071f)
- ✏️ #1860 [`db63fd3`](https://github.com/open-wa/wa-automate-nodejs/commit/db63fd3001d9220c40bc7e91da2b0c78695bd11b)

**Comment by smashah** [comment]
Thanks for your patience everyone. I have added some measures to ensure this does not happen again in a 3/4 months.
